lm75bd/lm75bd.c

```diff
@@ -26,9 +26,29 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
-  
-  return ERR_CODE_SUCCESS;
+    error_code_t errCode;  
+
+    uint8_t buf[2] = {0};   
+    uint8_t pointerRegister = 0x00;  
+
+    // Send a pointer to the temperature register
+    LOG_IF_ERROR_CODE(i2cSendTo(devAddr, &pointerRegister, 1));
+
+    // Read from the temperature register
+    LOG_IF_ERROR_CODE(i2cReceiveFrom(devAddr, buf, 2));
```

Comment:
These should be RETURN_IF_ERROR_CODE as it doesn't make sense to continue with either one if there was an error. 

Also replace the hard coded 1 and 2 with the sizeof() the buffers